v2/datastream-mongodb-to-firestore/pom.xml

```diff
@@ -68,7 +68,7 @@
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongodb-driver-sync</artifactId>
-            <version>5.6.1</version>
+            <version>5.3.0</version>
```

Comment:
Can we use a property here and in the core to keep in sync? I think someone tried to do that earlier with line 36, but it was never used possibly.  And also keep at 5.6.1? Thanks

Reply:
done